pages/login.js

```diff
@@ -0,0 +1,33 @@
+import useUser from "@/hooks/useUser";
+import StyledHeadlineTwo from "@/components/Layout/StyledHeadlineTwo";
+import ActionLink from "@/components/Layout/ActionLink";
+import ActionButton from "@/components/Layout/ActionButton";
+
+export default function LoginPage() {
+  const { login } = useUser();
+
+  function onSubmit(event) {
+    event.preventDefault();
+    login({
+      email: event.target.email.value,
+      roomNumber: event.target.roomNumber.value,
+    });
+  }
+  return (
+    <>
+      <StyledHeadlineTwo>Log In</StyledHeadlineTwo>
+      <form onSubmit={onSubmit}>
+        <label htmlFor="email">
+          Email Address:
+          <input id="email" name="email" type="email" required />
```

Comment:
If you are nesting your `input` element inside your `label` element, you do not need to connect your label and input element via `htmlFor` and `id`.
```suggestion
        <label>
          Email Address:
          <input name="email" type="email" required />
```